### ISSUE TITLE
bug: folder name with all numbers can be created, but not used for Python scripts

### ISSUE THREAD

**Issue by mhuang74** [windmill-labs/windmill#3623]
### Describe the bug

Python script fails to run after moving to a Folder named `12345`.

```
Traceback (most recent call last):
  File "<frozen runpy>", line 189, in _run_module_as_main
  File "<frozen runpy>", line 159, in _get_module_details
  File "<frozen importlib._bootstrap_external>", line 1074, in get_code
  File "<frozen importlib._bootstrap_external>", line 1004, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/tmp/windmill/wk-default-6c662666b99d-08sx8/018f1ca3-538c-bf6c-dad5-f4dbde3b13b9/wrapper.py", line 9
    from f.12345 import pause_all_campaigns as inner_script
          ^^^^^^
SyntaxError: invalid syntax
```

### To reproduce

1. Create any Python script under a User path. Execution succeeds.
2. Move the script to a folder named `12345`, and execution fails with Syntax Error
3. Move the script to a folder named `CID_12345`, and execution succeeds

### Expected behavior

Python script should run after moving to a valid folder.

### Browser information

Safari Version 17.3.1 (18617.2.4.11.12, 18617)

### Application version

Windmill CE v1.319.1

**Comment by rubenfiszel** [comment]
ETA to solve this end of week. Thanks for the report.

**Comment by rubenfiszel** [comment]
Fixed in cf6ffef48
